Makefile

```diff
@@ -32,7 +32,7 @@ build-all:
 
 .PHONY: build-node ## Build release node with `tee-dev` feature
 build-node:
-	cargo build --locked -p $(call pkgid, $(NODE_BIN)) --release --features=tee-dev
+	cargo build --locked -p $(call pkgid, $(NODE_BIN)) --release --features=tee-dev,fast-runtime
```

Comment:
This will have an impact on local build commands only, I don't think CI is affected where docker build is applied.

Reply:
This is really valuable comment thanks. Shall we build/push/pull specific docker build for parachain ts tests or from now on we will build fast-run in general first CI?

Reply:
We are safe to apply this flag to general CI.

The release docker image is ensured to be built with different flags.